### DISCUSSION TITLE
moose on hpc cluster

### DISCUSSION THREAD

**Discussion by rrezakhani** [idaholab/moose#16347] [Q&A Getting Started] [answered]
Hi Everyone, 

I am trying to install moose on psc cluster following this instruction:
https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html

I have completed petsc and libmesh compilations, but when I do make in test folder I get errors complaining about finding petsc:
![Screenshot from 2020-11-30 17-46-35](https://user-images.githubusercontent.com/43370625/100675636-4c38b480-3335-11eb-882b-92449ce16a86.png)

Do you have any suggestion?

Thanks a lot,
Roozbeh

**Comment by YaqiWang** [comment, marked as answer]
Assuming you have compiled PETSc properly. Can you check the configure log of libMesh carefully to see if it detected PETSc? You may compiled libMesh without PETSc. If that is the case, you may try telling libMesh configure where and which PETSc to use with something like `PETSC_DIR=[you petsc directory here] PETSC_ARCH=[the petsc build, could be something like arch-moose] ./scripts/update_and_rebuild_libmesh.sh`.

**Reply by rrezakhani** [reply]
Thanks Yaqi. There was a mistake on my side, there were some old compilation files left, which caused the problem. I redid everything based on the instruction page, and it works well.
